parlai/crowdsourcing/tasks/fast_acute/analysis.py

```diff
@@ -0,0 +1,743 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+ACUTE-Eval Analyzer.
```

Comment:
does this have feature parity with the current analysis script?

Reply:
Discussed offline - there are a few small things missing from the current analysis script. Will add in the most important of those before removing the old version, and will add a list of the others, as well as instructions on how to find the old version of the analysis script and how to file an issue to request adding back in any missing features.